### ISSUE TITLE
Dashboard Fatal Error after Upgrade from 3.3.4 to 4.0.0

### ISSUE THREAD

**Issue by curtisgray** [mautic/mautic#10463]
### Mautic Version

4.0.0

### PHP version

7.4

### What browsers are you seeing the problem on?

Microsoft Edge

### What happened?

After upgrading from 3.3.4 to 4.0.0, the Mautic Dashboard displays an "Uh oh" error. The latest error in the ./var/logs/mautic_prod-[date].php file is:

```
[2021-09-21 18:58:47] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function getCount() on int" at /var/www/html/brm/app/bundles/StatsBundle/Aggregate/Calculator.php line 187 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getCount() on int at /var/www/html/brm/app/bundles/StatsBundle/Aggregate/Calculator.php:187)"} []
```

### How can we reproduce this issue?

Step 1: Ensure the version of Mautic is v3.3.4
Step 2: Connect to your Mautic server via ssh
Step 3: Change to the Mautic installation (e.g. `/var/www/mautic/`)
Step 4: Run the command `path/to/php bin/console mautic:update:find`
Step 5: Run the command `path/to/php bin/console mautic:update:apply`
Step 6: Run the command `path/to/php php app/console cache:clear`
Step 7: Visit Mautic dashboard on the web

### Relevant log output

```shell
[2021-09-21 18:58:47] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function getCount() on int" at /var/www/html/brm/app/bundles/StatsBundle/Aggregate/Calculator.php line 187 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getCount() on int at /var/www/html/brm/app/bundles/StatsBundle/Aggregate/Calculator.php:187)"} []
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by RCheesley** [comment]
Hi @curtisgray thanks for raising the issue. Can you confirm whether you have tried to clear the cache after the update completed?

**Comment by curtisgray** [comment]
Yes, I have cleared the cache.

**Comment by curtisgray** [comment]
@RCheesley, this error has something to do with Dashboard Widgets. Apparently, the Calculator.php code referenced in the log file cannot handle a customized Dashboard with different Widgets than the default.

A workaround is to remove all Widgets from the Dashboard. But, since the Dashboard is not accessible, due to the reported error, the Widgets were removed directly via the MySQL database using the following method:

You will need your Mautic database name. Below it is named `[mautic_database]`.
Run the following command on your Mautic MySQL Database: `delete from [mautic_database].widgets;`. The steps I used are listed below.

1. Connect to the Mautic server via ssh
2. Connect to mysql. For example `mysql -pP@ssworDV4lue`
3. Run the following SQL command: `delete from [mautic_database].widgets;`

In my case the Dashboard worked after this.
